### ISSUE TITLE
Make sure we do not save unnecessary data with the model

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/gate-lf-pytorch-json#19]
Make sure we do not save stuff as the loaded validation set or similar with the model file.

**Comment by johann-petrak** [comment]
Validation set has been removed, but we still need to check for other transient fields and make sure all fields get saved/restored/re-calculated properly, leaving open.

**Comment by johann-petrak** [comment]
Check:
* cuda settings should always get set from parameter and device capabilities (model may get used on different HW than trained)
* Any fields that point to files may need updating if the model gets moved
